src/test/regress/sql/plpgsql.sql

```diff
@@ -1733,7 +1733,7 @@ begin
 		when data_exception then  -- category match
 			raise notice 'caught data_exception';
 			x := -1;
-		when NUMERIC_VALUE_OUT_OF_RANGE OR CARDINALITY_VIOLATION then
+		when NUMERIC_VALUE_OUT_OF_RANGE OR CARDINALITY_VIOLATION OR TOO_MANY_ROWS then
 			raise notice 'caught numeric_value_out_of_range or cardinality_violation';
```

Comment:
I suppose we should also extend the error message, appending `or too_many_rows` to it, although this creates more differences with upstream.

Reply:
My vote is for error message extending. That's right it creates more differences, but this is insignificant, imo.